Entity/CustomField.php

```diff
@@ -203,8 +204,11 @@ class CustomField extends FormEntity implements UniqueEntityInterface, UuidInter
 
     /**
      * @var bool
+     * @Groups({"custom_field:read", "custom_field:write", "custom_object:read", "custom_object:write"})
      */
-    private $isUniqueIdentifier = false;
+    private bool $isUniqueIdentifier = false;
+    
+    private bool $wasChangeIsUniqueIdentifier = false;
```

Comment:
```suggestion
    private bool $hasChangedIsUniqueIdentifier = false;
```

I'd has is recommended over was. has is used more often and while not relevant here but a lot of the reflection services in the symfony community will look for has but not for was. API Platform, Symfony serializers, Twig etc.